applications/GeoMechanicsApplication/tests/settlement_workflow.py

```diff
@@ -35,9 +35,28 @@ def test_d_settlement_workflow(self):
                 "DISPLACEMENT", times_to_check[i], expected_data, node_ids)
 
             self.assertEqual(len(actual_nodal_values), len(expected_nodal_values))
-            for (actual_displacement, expected_displacement) in zip(actual_nodal_values, expected_nodal_values):
+            for actual_displacement, expected_displacement in zip(actual_nodal_values, expected_nodal_values):
                 self.assertVectorAlmostEqual(actual_displacement, expected_displacement, 3)
 
+            if i > 2:
+                self.check_stress_values(expected_result_file_name, i, node_ids, reader, result_file_name,
+                                         times_to_check, "TOTAL_STRESS_TENSOR")
+                self.check_stress_values(expected_result_file_name, i, node_ids, reader, result_file_name,
+                                         times_to_check, "CAUCHY_STRESS_TENSOR")
+
+    def check_stress_values(self, expected_result_file_name, i, node_ids, reader, result_file_name, times_to_check,
+                            variable_name):
+        actual_data = reader.read_output_from(result_file_name)
+        actual_nodal_values = reader.nodal_values_at_time(variable_name, times_to_check[i], actual_data, node_ids)
```

Comment:
Since this method is intended to be used for stresses, we may reflect that in the variable names, too. E.g.:
```suggestion
        actual_nodal_stress_vectors = reader.nodal_values_at_time(variable_name, times_to_check[i], actual_data, node_ids)
```

Reply:
Good suggestion, done